components/reports/SubjectOverview.tsx

```diff
@@ -38,15 +38,20 @@ const CollectionLink = ({
   uri,
   collectionName,
   repoUrl,
+  omitQueryParams,
 }: {
   uri: string
   collectionName: string
   repoUrl: string
+  omitQueryParams?: string[]
 }) => {
   const pathname = usePathname()
   const searchParams = useSearchParams()
   const newUrl = new URLSearchParams(searchParams)
   newUrl.set('quickOpen', uri)
+  if (omitQueryParams?.length) {
+    omitQueryParams.forEach((param) => newUrl.delete(param))
+  }
```

Comment:
nit: could make this slightly shorter

```suggestion
omitQueryParams?.forEach((param) => newUrl.delete(param))
```